ProfileManager.js

```diff
@@ -446,26 +434,30 @@ export default class ProfileManager {
    * @param {object} options - The options to use.
    * @param {string} options.profileId - The ID of the profile to get a signer
    *   for.
-   * @param {string} options.capabilityInvoker - The type of invoker to sign
-   *   capability invocations.
+   * @param {boolean} options.useEphemeralSigner - Flag to enable invoking
+   *  capabilities with the ephemeral invocation signer, default `true`.
```

Comment:
I think we need more detail about this ... it's confusing here. We're going to use the ephemeral signer to do what? We're going to use it when invoking the profile's zcap invocation key -- instead of using the profile agent's zcap invocation key directly. We should also say why this option is here (it reduces the number of hits to the WebKMS by trading off the penalty for security there for an ephemeral key that can be used locally but will not be long lived to help keep a decent security profile). I think we need more details about this sort of thing so people can know what's going on.

Reply:
We should probably mention this in a doc section in one place and then just reference that. That way people can understand what this flag is about -- and why they really don't need to touch it ever unless they are doing something special.

Reply:
This may have been missed https://github.com/digitalbazaar/bedrock-web-profile-manager/pull/33#discussion_r443338177
> We will need some text around this to help bolster a developer's understanding of what an "ephemeral signer" is and why it is important.

I believe we should expand on this, but not in the jsdoc for the parameter. This should be in a readme or in a similar piece of supporting documentation.

Reply:
I think we should have it in the code -- and maybe not in the README... at least not highlighted very well in the README if we put it there... we don't want people to use it, right? So best if it's documented where experts will use it.

Reply:
Good point. Will document this alongside the class description.